ci/terraform/aws/kms.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_kms_key" "id_token_signing_key" {
+  description             = "KMS signing key for ID tokens"
```

Comment:
Do we want to parametrise the name for this with the environment, or are we sharing a single signing key for all environments?

Reply:
This would give us a key for non-prod and a key for prod as they'll be in different accounts. Unless we think we need a key for each environment?

Reply:
As I understand it, we have separate Redis and Dynamo instances for the different environment, seems to me it makes sense to keep it consistent and have separate signing keys for each environment?

Reply:
Ok if that's where things are at. It will need to be added to the alias. Let me push something up

Reply:
Should be one for each env now @mrwilson